src/System.Runtime.Serialization.Formatters/tests/TargetFrameworkMoniker.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+namespace System.Runtime.Serialization.Formatters.Tests
+{
+    // The values represent platforms where there was change in the serialization for one or more types.
+    public enum TargetFrameworkMoniker
+    {
+        netfx461,
+        netfx471,
+        netfx472,
+        netcoreapp20,
+        netcoreapp21,
+    }
```

Comment:
Don't we have a type named the same thing in another test namespace? Can we just use that one?  Or if not, can we pick a different name?

Reply:
Currently not as this is the only place where we are versioning .NET Framework behaviors. If we need it somewhere else it makes sense to put it into Common or CoreFx.Private.TestUtilities.

Reply:
Why not just add the additional needed definitions to https://github.com/dotnet/buildtools/blob/5c19d590d5317c9f3b9813e18787b27c3a7ebfac/src/xunit.netcore.extensions/TargetFrameworkMonikers.cs#L10?

Reply:
IMO this doesn't belong into a xunit namespace and has the downside of waiting for a new buildtools update when we quickly need to add a new tfm like in this case here. But I'm certainly open for suggestions how to converge these two files without the mentioned issues.